pkgs/applications/radio/yaac/default.nix

```diff
@@ -0,0 +1,34 @@
+{ makeWrapper, lib, stdenv, fetchurl, unzip, jre8 }:
+
+stdenv.mkDerivation rec {
+  version = "1.0-beta181";
+  pname = "YAAC";
+
+  src = fetchurl {
```

Comment:
How about using `fetchzip` here?

Reply:
Updated, didn't use initially as wasn't aware. Was tricky with the stripRoot at first. but got it working.